R/assign_nearest_land.R

```diff
@@ -23,20 +23,20 @@ assign_nearest_land <- function(dat_object,
   message("dat_object object must contain columns 'longitude' and 'latitude'.")
 
   # ensure that data object is spatial data frame
-  spat_dat <- st_as_sf(dat_object)
+  sf::spat_dat <- st_as_sf(dat_object)
```

Comment:
I think you mean:

`spat_dat <- sf::st_as_sf(dat_object)`

Reply:
mmmhmmm